domenetjenester/arbeidsforhold/src/main/java/no/nav/foreldrepenger/domene/iay/modell/OppgittOpptjening.java

```diff
@@ -39,6 +39,14 @@ public OppgittOpptjening(UUID eksternReferanse) {
         // å lagring
         setOpprettetTidspunkt(LocalDateTime.now());
     }
+    public OppgittOpptjening(OppgittOpptjening oppgittOpptjening) {
+        this.uuid = oppgittOpptjening.uuid;
```

Comment:
Bør kanskje lage ny UUID for ny instans. Men litt usikker om vi trenger uuid overhodet.